doorstop/core/item.py

```diff
@@ -664,7 +665,8 @@ def _get_issues_both(self, document, tree, skip):
         if settings.CHECK_CHILD_LINKS and self.normative:
             items, documents = self._find_child_objects(document=document,
                                                         tree=tree,
-                                                        find_all=False)
+                                                        find_all=True)
```

Comment:
Should find_all only be run if CHECK_CHILD_LINKS_STRICT is true to speed things up?